src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx

```diff
@@ -108,7 +108,7 @@ const GroupDetails = createReactClass({
         // https://github.com/reactjs/react-router/blob/v2.0.1/modules/index.js#L25
         if (this.props.params.groupId != data.id) {
           const location = this.props.location;
-          return void browserHistory.push(
```

Comment:
we don't need this return? seems like other code will execute that didn't before?

Reply:
Ah, thank you